_data/people/cvarni.yml

```diff
@@ -0,0 +1,6 @@
+name: Carlo Varni
+title:
```

Comment:
@CarloVarni can you also add your title at UC Berkeley and your email? 

c.f. https://github.com/iris-hep/iris-hep.github.io-source/blob/f6e4d0202554c74b3fa661b0e54c711486b088f2/_data/people/matthewfeickert.yml#L1-L8

as an example.

Reply:
Hi @matthewfeickert, nice to see you again! 
I've added both the title and the e-mail

Reply:
Hi @matthewfeickert, nice to see you again. I have added both the title and the Mail.